tests/cloudpickle_test.py

```diff
@@ -44,11 +44,15 @@
 
 import cloudpickle
 from cloudpickle.compat import pickle
-from cloudpickle.cloudpickle import _is_importable
+from cloudpickle.cloudpickle import _is_importable, _PICKLE_BY_VALUE_MODULES
 from cloudpickle.cloudpickle import _make_empty_cell, cell_set
 from cloudpickle.cloudpickle import _extract_class_dict, _whichmodule
 from cloudpickle.cloudpickle import _lookup_module_and_qualname
+from cloudpickle.cloudpickle import register_pickle_by_value
+from cloudpickle.cloudpickle import unregister_pickle_by_value
+from cloudpickle.cloudpickle import is_registered_pickle_by_value
```

Comment:
nitpick: `register_pickle_by_value` (and the two other utilities) will now be part of the `cloudpickle` public API, and should be directly importable using. 

```suggestion
from cloudpickle import register_pickle_by_value
from cloudpickle import unregister_pickle_by_value
from cloudpickle import is_registered_pickle_by_value
```